recording/record.js

```diff
@@ -50,14 +50,14 @@ const transcodeStreamToOutput = spawn('ffmpeg',[
         '-maxrate', `${VIDEO_BITRATE}`,
         '-g', `${VIDEO_GOP}`,
     // apply a fixed delay to the audio stream in order to synchronize it with the video stream
-    '-filter_complex', 'adelay=delays=1000|1000',
+    -filter_complex', 'adelay=delays=500|500, aresample=async=1:first_pts=0',
```

Comment:
There's a leading quote missing here

Reply:
[record.js.zip](https://github.com/aws-samples/amazon-chime-sdk-recording-demo/files/7039636/record.js.zip) I just reloaded this modified record.js in a new recording demo container and it ran fine.  Here are the differences.
![image](https://user-images.githubusercontent.com/72471320/130629464-8e740262-2bfe-4a33-9f3d-2b28f7d6ac8c.png)  As far as I can tell, we just lost a leading quote somehow when it was submitted.  Let me know what I need to do to correct it.

Reply:
If you can update the branch that you submitted the PR from (https://github.com/albrich-me/amazon-chime-sdk-recording-demo/tree/patch-1 ), the pull request should get updated automatically